### PR TITLE
Add support to specify logging levels

### PR DESCRIPTION
**Pull request by artem-cliqz**

### CODE REVIEW

File: graylog_hook.go [hunk 1]
```diff
@@ -40,6 +41,12 @@ type graylogEntry struct {
 	line int
 }
 
+func NewGraylogLevelHook(addr string, extra map[string]interface{}, levels []logrus.Level) *GraylogHook {
+	hook := NewGraylogHook(addr, extra)
+	hook.levels = levels
```
Comment (dbalan) [inline graylog_hook.go]: I'd do it how `logrus.SetLevel` does it.

```golang
  // Only log the X severity or above.
  log.SetLevel(log.XLevel)
```

`NewGraylogLevelHook(addr, extra, minLevel logrus.Level)`

**Comment by artem-cliqz** [comment]
@dbalan Fixed, could you have a look?

File: graylog_hook.go [hunk 2]
```diff
@@ -40,6 +41,19 @@ type graylogEntry struct {
 	line int
 }
 
+func NewGraylogLevelHook(addr string, extra map[string]interface{}, level logrus.Level) *GraylogHook {
+	levels := []logrus.Level{}
```
Comment (dbalan) [inline graylog_hook.go:45]: `NewGrayLogHookWIthLevel` makes more sense.

**Review by dbalan — APPROVED** [review]
LGTM.

**Pull request merged**